### PR TITLE
Docs: Added HMC setup and HMC userid reqs

### PR DESCRIPTION
**Pull request by andy-maier**
For details, see the commit message.

Notes for reviewers:
* Note that the .rst files for the zhmc_* modules are generated, so make your comments on the source of the information which is in the .py source files.
* It would be good to double check the module-specific task and object-access permissions.

### CODE REVIEW

File: plugins/modules/zhmc_adapter.py [hunk 1]
```diff
@@ -37,14 +37,19 @@
...
 requirements:
-  - Access to the WS API of the HMC of the targeted Z system
-    (see :term:`HMC API`).
   - The targeted Z system must be in the Dynamic Partition Manager (DPM)
     operational mode.
+  - "The HMC userid must have these task permissions:
+    'Adapter Details', 'Create HiperSockets Adapter',
+    'Delete HiperSockets Adapter', 'Configure Storage - System Programmer'."
+  - "The HMC userid must have object-access permissions to these objects:
+    Target adapters, CPCs of target adapters."
```
Comment (EdGue42) [inline plugins/modules/zhmc_adapter.py:52]: No single ticks here on purpose I guess?

File: plugins/modules/zhmc_cpc.py [hunk 2]
```diff
@@ -34,17 +34,22 @@
...
-  - Deactivate a CPC (Z system).
-  - Activate a CPC and update its properties.
+  - Deactivate/Stop a CPC (Z system).
+  - Activate/Start a CPC and update its properties.
   - Gather facts about a CPC, and for DPM operational mode, including its
     adapters, partitions and storage groups.
   - Update the properties of a CPC.
 author:
   - Andreas Maier (@andy-maier)
   - Andreas Scheuring (@scheuran)
```
Comment (EdGue42) [inline plugins/modules/zhmc_cpc.py:44]: wondering: would it make sense to scrap such lists at some point, to remove people no longer involved?

**Review by EdGue42 — APPROVED** [review]

File: plugins/modules/zhmc_adapter.py (hunk 1 above)
Comment (andy-maier) [reply plugins/modules/zhmc_adapter.py:52]: Yes, the object access permissions in this case are not names, like the tasks are.

File: plugins/modules/zhmc_cpc.py (hunk 2 above)
Comment (andy-maier) [reply plugins/modules/zhmc_cpc.py:44]: Not sure what rules we should employ on that. Let's discuss separately.

**Pull request merged**